docs/changelog.rst

```diff
@@ -2,6 +2,8 @@
 Changelog
 =========
 
+* :bug:`2773` Timestamps will be correctly read for trades in the Kraken exchange.
```

Comment:
changelog is fine. Just please remember to edit the FTX one as I told you in the other PR.